docs/data/material/components/dividers/VerticalDividerMiddle.js

```diff
@@ -16,7 +16,7 @@ export default function VerticalDividerMiddle() {
         '& svg': {
           m: 1,
         },
-        '& hr': {
+        '& .MuiDivider-root': {
```

Comment:
I'm not familiarized with the styling patterns in demos. Not sure this is something we want to do.